config/testconfig.js

```diff
@@ -33,12 +33,13 @@ config["leagues"]["lonewolf"]["heltour"]["baseEndpoint"] = "https://staging.lich
 config["leagues"]["45+45"]["scheduling"]["channel"] = "unstable_bot";
 config["leagues"]["45+45"]["results"]["channel"] = "unstable_bot";
 config["leagues"]["45+45"]["gamelinks"]["channel"] = "unstable_bot";
-config["leagues"]["45+45"]["alternate"]["channel_id"] = "G2G6URBT5";
+config["leagues"]["45+45"]["alternate"]["channel_id"] = "C0VCCPMJ8";
 config["leagues"]["lonewolf"]["scheduling"]["channel"] = "unstable_bot-lonewolf";
 config["leagues"]["lonewolf"]["results"]["channel"] = "unstable_bot-lonewolf";
 config["leagues"]["lonewolf"]["gamelinks"]["channel"] = "unstable_bot-lonewolf";
 
 config["channel_map"]["G2G6URBT5"] = "45+45";
```

Comment:
So I suggest reverting all changes to this file (and eventually putting them to separate PR).